HW_1/task_2.py

```diff
@@ -0,0 +1,22 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов.
+# Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок,
+# если известно, что Петя и Сережа сделали одинаковое количество журавликов,
+# а Катя сделала в два раза больше журавликов,
+# чем Петя и Сережа вместе?
+
+# *Пример:*
+
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+
+n = int(input("Введите количество журавликов: "))
+if n // 6 and n % 2 == 0:
```

Comment:
Если резюмировать, то 
```suggestion
if n % 6 == 0:
```